pages/slider/permutations.page.tsx

```diff
@@ -96,7 +177,12 @@ export default function SliderPermutations() {
       <ScreenshotArea disableAnimations={true}>
         <PermutationsView
           permutations={permutations}
-          render={permutation => <Slider ariaLabel="Slider" i18nStrings={i18nStrings} {...permutation} />}
+          render={(permutation, index) => (
+            <Box>
```

Comment:
Is it OK to include the box and the label in the permutation screenshot?

Reply:
That does look ok to me, as these components don't interfere with the slider. One disadvantage is that a change to the box styles would cause these permutations to fail, but that is true for the most of our screenshot tests anyways.